app/views/vms/index.html.erb

```diff
@@ -1,248 +1,20 @@
-            <% end %>
-          </ul>
-        <% end %>
-        <% if current_user.admin? %>
-          <%= link_to '',
-                      {controller: :vms, action: 'revive_vm', id: vm.name},
-                      method: :post,
-                      class: "btn-manage float-right checkmark" %>
-
-        <% end %>
-      </td>
-      <td class="d-none"> <!-- Also not intended to be shown! -->
-        <%= vm.config&.description %>
-      </td>
-    </tr>
-  <% end %>
-  </tbody>
-</table>
-
-<h2 id="archived">Archived machines</h2>
-
-<input class="filter-box" type="text" id="filter_archived" onkeyup="filter_table('filter_archived', 'vms_archived', document)" placeholder="Search">
-
-<table data-toggle="table" class="table" id="vms_archived">
-  <thead>
-  <tr>
-    <th scope="col" data-sortable="true" class="rounded-left">Name</th>
-    <th scope="col" data-sortable="true">Project</th>
-    <th scope="col" data-sortable="true" class="rounded-right">Responsible</th>
-    <th scope="col" class="d-none">Description</th> <!-- This is not intended to be shown, just important for the search functionality -->
-  </tr>
-  </thead>
-  <tbody>
-  <% @archived_vms&.each do |vm| %>
-    <tr>
-      <td class="table-active">
-        <%= link_to vm.name,
-                    {controller: :vms, action: 'show', id: vm.name},
-                    method: :get, class: "btn btn-link"  %>
-      </td>
-      <td class="table-active">
-        <% if vm.project.nil? %>
-          No project found!
-        <% else %>
-          <%= link_to vm.project.name, vm.project %>
-        <% end %>
-      </td>
-      <td class="table-active align-middle">
-        <% if vm.responsible_users.empty? %>
-          No responsible users found!
-        <% else %>
-          <ul class="list-unstyled">
-            <% vm.responsible_users.each do |user| %>
-              <li><%= link_to user.name, user %></li>
-            <% end %>
-          </ul>
-        <% end %>
-        <% if current_user.admin? %>
-          <%= link_to '',
-                      {controller: :vms, action: 'revive_vm', id: vm.name},
-                      method: :post,
-                      class: "btn-manage float-right arrow-up" %>
-        <% else %>
-          <%= link_to '',
-                      {controller: :vms, action: 'request_vm_revive', id: vm.name},
-                      method: :post,
-                      class: "btn-manage float-right arrow-up" %>
-        <% end %>
-      </td>
-      <td class="d-none"> <!-- Also not intended to be shown! -->
-        <%= vm.config&.description %>
-      </td>
-    </tr>
-  <% end %>
-  </tbody>
-</table>
+<!-- Minified script sources for sortable tables -->
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.12.1/bootstrap-table.min.css"
+      xmlns="http://www.w3.org/1999/html">
+<script src="//cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.12.1/bootstrap-table.min.js"></script>
```

Comment:
Is this the only place where this JS is needed? Can it be moved to the `<head>`?

Reply:
It is already moved to the _footer
This was readded during merge

Reply:
:+1: